optimism/src/keccak/constraints.rs

```diff
@@ -69,7 +71,7 @@ impl<F: Field> KeccakInterpreter<F> for Env<F> {
         // Despite `KeccakWitness` containing both `curr` and `next` fields,
         // the Keccak step spans across one row only.
         Expr::Atom(ExprInner::Cell(Variable {
-            col: column,
+            col: column.to_column(),
```

Comment:
In order to be able to use the generic prover, the expressions must be given as generic columns. Thus, variable columns must be mapped to the `X(i)` notation, which is performed thanks to the `ColumnIndexer` trait implementation.